src/templates/antora/template.ts

```diff
@@ -162,11 +176,11 @@ const getMemberPages = (root: Api, templateDelegate: HandlebarsTemplateDelegate,
   data.keywords = data.keywords.join(', ');
   return [{
     type: 'json',
-    filename: createFileName(data, 'json'),
+    filename: ('_data/api/' + data.type + '_' + data.fullName.replace(/\./g, '_') + '.json').toLowerCase(),
```

Comment:
Why was this removed? The old version was more readable and it could have been updated to delegate to the `AntoraTemplate.getFilePath` Instead of inlining it like this.